project.godot

```diff
@@ -170,11 +170,6 @@ toggle_FPS={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777246,"unicode":0,"echo":false,"script":null)
  ]
 }
-g_hold_forward={
```

Comment:
I'm pretty sure this and the RubySetupSystem changes are actually backwards, so these changes should be undone.